sections/02_intro.md

```diff
@@ -1,3 +1,25 @@
 ## Introduction
 
-Intro goes here.
+### Potential writing prompt
+
+One potential future that we could imagine is a world in which data, once
```

Comment:
I definitely agree with the three subcategories and the content of this stub. 

But, for consistency, can you mention them either before or after the period? Right now I am not sure which sentences `(CATEGORIZE)` or `(TREAT)` are referencing